design/scenarios.md

```diff
@@ -0,0 +1,73 @@
+# Working Scenarios of Kata Container
+
+- [CRI](#cri)
+- [OCI based CRI Runtimes](#oci-based-cri-runtimes)
+  - [PodSandbox in OCI based runtimes](#podsandbox-in-oci-based-runtimes)
+  - [Typical working pattern of CRI runtimes](#typical-working-pattern-of-cri-runtimes)
+- [Frakti CRI Runtime](#frakti-cri-runtime)
+  - [The APIs Frakti rely on](#the-apis-frakti-rely-on)
+  - [Working pattern of Frakti](#working-pattern-of-frakti)
+- [Working with Docker](#working-with-docker)
+
+## CRI
+
+The operations defined in CRI RuntimeService are quite straightforward. There are two parts (as shown in the figure): PodSandbox operations and Container Operations.
```

Comment:
Could you put backticks around all API calls, types and commands/processes (such as `RuntimeService`)?